.github/workflows/publish-docker-image.yml

```
@@ -140,6 +140,8 @@ jobs:
 
       - name: Check multi-arch manifest
         run: |
+          cat architectures_actual.txt
           cat supported-architectures.txt | sort | uniq > architectures_expected.txt
           # we're counting the entries only, because windows os.version numbers change too frequently
-          diff $(cat architectures_expected.txt | wc -l) $(cat architectures_actual.txt | wc -l)
+          # exclude manifest entries with "unknown" architectures
+          diff <(echo "$(cat architectures_expected.txt | wc -l)") <(echo "$(cat architectures_actual.txt | grep -v unknown_unknown_ | wc -l)")
```

So we want to just check the lines or the content? If the latter:
```suggestion
          diff <(echo "$(cat architectures_expected.txt | sort )") <(echo "$(cat architectures_actual.txt | grep -v unknown_unknown_ | sort)")
```

Thoughts?

---

See the hint at the previous PR #120: for Windows, the `os.version` changes frequently, so comparing the content would make the comparison fragile. That's why checking only the count is used as indication whether all Linux and Windows architectures are included in the final image.

---

Thanks for resolving my doubts 🙇